### PR TITLE
Initial commit about page

### PR DESCRIPTION
**Pull request by priyajit27**
ADD: About Page

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,17 +1,14 @@
...
   return (
     <>
-      <Navbar />
+      {/* <Navbar />
       <Routes>
         <Route path="/" element={<Home />} />
         <Route path="*" element={<Error />} />
-      </Routes>
+      </Routes> */}
+      <About />
```
Comment (codadept) [inline src/App.jsx]: Why have you removed all the code and only displayed About Us page? You should have created a new Route instead

```jsx
<Route path="/about" element={<About />} />
```

File: src/Pages/About/About.module.scss [hunk 2]
```diff
@@ -0,0 +1,171 @@
+* {
+  background-color: #ffcb81;
+}
```
Comment (codadept) [inline src/Pages/About/About.module.scss]: Don't give background colour to * i.e. everything. Instead give it to the topmost class of your about page

**Review by codadept — CHANGES_REQUESTED** [review]

**Comment by codadept** [comment]
image

Also, the Durga Maa's image along with the Trishul is stretched. Fix it too

**Review by username-naz — CHANGES_REQUESTED** [review]
image
center align the text
image
fix this diamond

**Pull request closed without merging**